### ISSUE TITLE
Build failed with react native 0.69 (Android)

### ISSUE THREAD

**Issue by marcoburrometo** [react-native-menu/menu#394]
Hello! 
I updated react native to 0.69.1 version but build fails on android.

I have this error:
`Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1`ù

Can someone help me?

**Comment by arjendevos** [comment]
It looks like the expo build.gradle the kotlinVersion is not specified, if we override this manually to 1.6.0 it works

**Comment by robertherber** [comment]
@arjendevos How and where do we do this? Is it in the expo package or menu package?

**Comment by robertherber** [comment]
@arjendevos Thanks!

Did you set it up like this, or did you configure anything else?
```json
[
  "expo-build-properties",
  {
    "android": {
      "kotlinVersion": "1.6.10"
    }
  }
]
```

**Comment by arjendevos** [comment]
@robertherber This should work

**Comment by emberist** [comment]
Same issue here, but not with expo

**Comment by emberist** [comment]
@robertherber I fixed the issue specifying a Kotlin version

Following this https://github.com/react-native-webview/react-native-webview/issues/2578#issuecomment-1177372363

**Comment by kahx** [comment]
> @robertherber I fixed the issue specifying a Kotlin version
> 
> Following this [react-native-webview/react-native-webview#2578 (comment)](https://github.com/react-native-webview/react-native-webview/issues/2578#issuecomment-1177372363)
> 
> EDIT
> 
> The project build successfully, but the issues is not solved

Same...

**Comment by Naturalclar** [comment]
Should be fixed in 0.6.0 without modifying your own build.gradle 😄
